crates/iota/src/iota_commands.rs

```diff
@@ -73,6 +73,7 @@ const DEFAULT_FAUCET_PORT: u16 = 9123;
 const DEFAULT_GRAPHQL_PORT: u16 = 9125;
 #[cfg(feature = "indexer")]
 const DEFAULT_INDEXER_PORT: u16 = 9124;
+const DEFAULT_COMMITTEE_SIZE: usize = 1;
```

Comment:
It seems like we had those constants sorted alphabetically, just in case you care about it.

Reply:
Not sure it was on purpose but I changed it anyway https://github.com/iotaledger/iota/pull/7852/commits/0caedf3824353d010e0bcb6e0f3e38590e5d5829